### ISSUE TITLE
Too many arguments on the Message class are marked as Awaitable

### ISSUE THREAD

**Issue by johman10** [youtype/mypy_boto3_builder#171]
**Describe the bug**
After the fix on issue https://github.com/youtype/mypy_boto3_builder/issues/168, it seems like there are some attributes that are marked as awaitable, while they shouldn't be.

This goes for `receipt_handle` and `queue_url`. I wrote some code that proved that those are the only Awaitables.

**To Reproduce**

1. Install `types-aiobotocore[essential]@2.4.0.post1`
2. Run `mypy` on the following code sample

```py
from types_aiobotocore_sqs.service_resource import Message

async def process_message(message: Message):
    handle = message.receipt_handle  # this is correct code
    queue_url = message.queue_url  # this is correct code
```

**Actual output**

```
error: Incompatible types (expression has type "Awaitable[str]", TypedDict item "ReceiptHandle" has type "str")
```

**Expected output**
No errors

**Additional context**
Linux
aioboto3@10.1.0
Poetry

**Comment by vemel** [comment]
Hello. Thank you for the report.

I will try to find a way to unmark non-awaitable attributes.

**Comment by vemel** [comment]
Finally, it is fixed in `types-aiobotocore 2.4.2.post2`. Please test it and let me know if it works as it should.

I checked `aioboto3` code and found out that only non-identifier and non-reference attributes are `Awaitable`.

**Comment by johman10** [comment]
@vemel it seems to be working! Thanks a lot for the support. Closing the issue now.
